ddtrace/provider.py

```diff
@@ -38,8 +38,8 @@ class DefaultContextProvider(BaseContextProvider):
     thread-local storage. It is suitable for synchronous programming and
     Python WSGI frameworks.
     """
-    def __init__(self):
-        self._local = DefaultContextManager()
+    def __init__(self, reset_context_manager=True):
```

Comment:
Should this default be `True`?

Reply:
The default behavior up till now has been to always create a new thread-local storage `_local` attr when instantiating a `DefaultContextProvider`:

https://github.com/DataDog/dd-trace-py/blob/0.27/ddtrace/context.py#L219-L227

This default behavior does not present a problem for the existing initialization of `ThreadLocalContext` in a `TracerStackContext`.